src/libraries/System.Linq/README.md

```diff
@@ -0,0 +1,10 @@
+# System.Linq
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.linq.
```

Comment:
A few words about what is included in the library would be nice, even though most people are familiar with what Linq represents.

Reply:
honestly I copy & pasted this from System.Reflection.Metadata, should we do this in some files and not in others. Is this doc even useful without a single sample?

Reply:
I would try to include a short description of what the project contains rather than attempt to document it. In this case it would be something along the lines of "defines Linq extension methods for IEnumerable", vs. System.Linq.Queryable containing the equivalent extension methods for IQueryable.